test/sing_for_needs_web/schema/query/artists_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule SingForNeeds.Schema.Query.ArtistsTest do
+  @moduledoc false
+  use SingForNeedsWeb.ConnCase, async: true
+
+  @query """
+  {
+    artists {
+      name
+    }
+  }
+  """
+  test "artists query returns all artists" do
+    artists_fixture()
+
+    conn = build_conn()
+    conn = get conn, "/api", query: @query
+
+    assert %{"data" => %{
+      "artists" => [
+        %{"name" => "Artist 1"},
+        %{"name" => "Artist 2"},
+        %{"name" => "Artist 3"}
+      ]
+      }
+    } = json_response(conn, 200)
```

Comment:
This test is insufficient and it's not catching the broken structure of your `test/support/test_helpers.ex` `artists_fixture`, that you refactored for Ebert.

Maybe look at the deleted `test/sing_for_needs_web/controllers/artist_controller_test.exs` for guidance on how to improve it or add more tests.